src/style.css

```diff
@@ -1,3 +1,75 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+}
```

Comment:
Don't fix container size (if there is no such a requirement). Here, the width is set to 100%, which might not be necessary if the container is a block-level element, as it will naturally occupy the full width of its parent.